src/common_swagger_api/routes.clj

```diff
@@ -1,4 +1,10 @@
-(ns common-swagger-api.routes)
+(ns common-swagger-api.routes
+  (:require [clojure.java.io :as io]
+            [compojure.api.meta]))
+
+(defmethod compojure.api.meta/restructure-param :description-file
+  [_ path acc]
+  (update-in acc [:swagger] assoc :description (slurp (io/resource path))))
```

Comment:
Since this is a notable new feature to this lib, I think I'll also cut a `2.10.0` release with this function, then merge the rest of this PR in a `2.10.1-SNAPSHOT` version.